components/table/table-wrapper.js

```diff
@@ -243,11 +245,16 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				--d2l-table-border-radius: 0.3rem;
 				--d2l-table-border-radius-sticky-offset: calc(1px - var(--d2l-table-border-radius));
 				--d2l-table-cell-height: 41px; /* min-height to be 62px including border */
-				--d2l-table-cell-padding: 0.5rem 1rem;
+				--d2l-table-cell-padding: 0.5rem 15px;
 				--d2l-table-cell-padding-alt: calc(0.5rem - 1px) 1rem 0.5rem 1rem;
 				--d2l-table-header-background-color: var(--d2l-color-regolith);
 				--d2l-table-row-border-color-selected: var(--d2l-color-celestine);
 				--d2l-table-row-background-color-selected: var(--d2l-color-celestine-plus-2);
+				--d2l-sortable-button-border-radius: 0;
+				--d2l-sortable-button-vertical-padding: 1rem;
+				--d2l-sortable-button-height: 50px;
```

Comment:
Is this in the spec? I didn't see it. From the spec it looks like we'd just want to have top and bottom padding of 1.5rem. If you haven't yet, I'd recommend testing the table without the other types of cells in the header row to see what the height looks like if there were only the sort buttons (i.e., just delete the header cells that are the `fruits.map`)